### ISSUE TITLE
Non-text colour contrast for WCAG 2.1

### ISSUE THREAD

**Issue by davidhunter08** [nhsuk/nhsuk-frontend#473]
## What is it

The current link :focus style is failing WCAG 2.1 - 1.4.11 Non-Text Contrast (AA).

https://www.w3.org/WAI/standards-guidelines/wcag/new-in-21/#1411-non-text-contrast-aa

<img width="666" alt="Screenshot 2019-06-21 at 10 13 34" src="https://user-images.githubusercontent.com/14331000/59912446-ed67cf80-940d-11e9-93a9-405899a3364c.png">

**Comment by AdamChrimes** [comment]
This was released in [3.0.0](https://github.com/nhsuk/nhsuk-frontend/blob/master/CHANGELOG.md#300---7-november-2019)
